README.rst

```diff
@@ -0,0 +1,3824 @@
+given order.
+
+Note that, when this option is not set, all the available ciphers will
+be offered to the Hazelcast members with their default order.
+
+The following is an example configuration:
+
+.. code:: python
+
+   client = hazelcast.HazelcastClient(
+       ssl_ciphers="DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA"
+   )
+
+8.1.3. Mutual Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As explained above, Hazelcast members have key stores used to identify
+themselves (to other members) and Hazelcast clients have trust stores
+used to define which members they can trust.
+
+Using mutual authentication, the clients also have their key stores and
+members have their trust stores so that the members can know which
+clients they can trust.
+
+To enable mutual authentication, firstly, you need to set the following
+property on the server side in the ``hazelcast.xml`` file:
+
+.. code:: xml
+
+   <network>
+       <ssl enabled="true">
+           <properties>
+               <property name="javax.net.ssl.mutualAuthentication">REQUIRED</property>
+           </properties>
+       </ssl>
+   </network>
+
+You can see the details of setting mutual authentication on the server
+side in the `Mutual Authentication
+section <https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#mutual-authentication>`__
+of the Hazelcast IMDG Reference Manual.
+
+On the client side, you have to provide ``ssl_cafile``, ``ssl_certfile``
+and ``ssl_keyfile`` on top of the other TLS/SSL configurations. See the
+`TLS/SSL for Hazelcast Python
+Clients <#812-tlsssl-for-hazelcast-python-clients>`__ for the details of
+these options.
+
+9. Development and Testing
+==========================
+
+If you want to help with bug fixes, develop new features or tweak the
+implementation to your application’s needs, you can follow the steps in
+this section.
+
+9.1. Building and Using Client From Sources
+-------------------------------------------
+
+Follow the below steps to build and install Hazelcast Python client from
+its source:
+
+1. Clone the GitHub repository
+   (https://github.com/hazelcast/hazelcast-python-client.git).
+2. Run ``python setup.py install`` to install the Python client.
+
+If you are planning to contribute, please make sure that it fits the
+guidelines described in
+`PEP8 <https://www.python.org/dev/peps/pep-0008/>`__.
+
+9.2. Testing
+------------
+
+In order to test Hazelcast Python client locally, you will need the
+following:
+
+-  Java 8 or newer
+-  Maven
+
+Following commands starts the tests according to your operating system:
+
+.. code:: bash
+
+   bash run-tests.sh
+
+or
+
+::
+
+   PS> .\run-tests.ps1
+
+Test script automatically downloads ``hazelcast-remote-controller`` and
+Hazelcast IMDG. The script uses Maven to download those.
+
+10. Getting Help
+================
+
+You can use the following channels for your questions and
+development/usage issues:
+
+-  This repository by opening an issue.
```

Comment:
Probably, we should add Hazelcast Slack (https://slack.hazelcast.com) into this list